### ISSUE TITLE
NTR: Granulation tissue of skin or mucosa (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8004]
New term request Granulation tissue of skin or mucosa (elthat_100807191834):   UMLS: 0574734

**Comment by pnrobinson** [comment]
@MickeySegal
this is not nececssarily an abnormal finding if it results from a small wound etc. 
What is the intended use case?

**Comment by MickeySegal** [comment]
It is mostly used for various forms of epidermolysis bullosa

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000956
- label: Exuberant granulation tissue
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:28492007
- def: A manifestation of abnormal wound healing in which there is formation of excessive granulation tissue, with the granulation tissue rising above the level of surrounding skin and creating a raised, reddish, moist, and bumpy surface.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8004
- synonyms: Overgranulation(NONE)|Hypergranulating wound(NONE)
